sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md

```diff
@@ -28,6 +28,66 @@ For highly sensitive data, clients should consider additional layers of protecti
+
+### Configuration Options
+Azure Spring Boot Key Vault Starter deprecates all legacy properties of which the prefix is `azure.keyvault` and uses `spring.cloud.azure.keyvault` instead.
+When a deprecated property is detected while its active property is not found, the active property will be configured into application environment with value from the deprecated property.
+
+If you load configuration properties from Azure Key Vault, the preceding detection and replacement are also applicable for Key Vault property sources. Replaced properties from Key Vault have higher priorities than local ones.
+Note that replaced properties will not be refreshed as common properties from Key Vault property source.
+
+We also provide unified configuration properties that are applicable for Azure Spring Starters. When Key Vault properties are not configured, associated unified Azure Spring properties will take effects.
+#### Key Vault properties
+##### Active Properties
+|Name|Description|Default Value|Comment|
+|:---|:---|:---|:---
+spring.cloud.azure.keyvault.case-sensitive-keys|Defines the constant for the property that enables/disables case sensitive keys.|false||
+spring.cloud.azure.keyvault.credential.client-certificate-password|Password of the certificate file to use when performing service principal authentication with Azure.||||
+spring.cloud.azure.keyvault.credential.client-certificate-path|Path of a PEM certificate file to use when performing service principal authentication with Azure.||||
+spring.cloud.azure.keyvault.credential.client-id|Client id to use when performing service principal authentication with Azure.||||
+spring.cloud.azure.keyvault.credential.client-secret|Client secret to use when performing service principal authentication with Azure.||||
+spring.cloud.azure.keyvault.credential.tenant-id|Tenant id for the Azure resources.|||
+spring.cloud.azure.keyvault.enabled|To turn on/off Azure Key Vault Secret property source.|true||
+spring.cloud.azure.keyvault.environment.authority-host|Authority Host URI|https://login.microsoftonline.com/||
+spring.cloud.azure.keyvault.order|Define the order of the key vaults you are delivering (comma delimited, e.g 'my-vault, my-vault-2').|||
+spring.cloud.azure.keyvault.refresh-interval|Interval for PropertySource to refresh secret keys|1800000(ms)||
+spring.cloud.azure.keyvault.secret-keys|If application using specific secret keys. If this property is set, application will only load the keys in the property and won't load all the keys from keyvault, that means if you want to update your secrets, you need to restart the application rather than only add secrets in the keyvault.|||
+spring.cloud.azure.keyvault.secret-service-version|Valid secret-service-version value can be found [here][version_link].|The latest value||
+spring.cloud.azure.keyvault.uri|Azure Key Vault Uri.|||
+
+Note: for multiple Key Vault usage, specify your Key Vault name after the prefix of `spring.cloud.azure.keyvault`.
```

Comment:
It would be nice to have an example of what you mean here, otherwise it's not clear if people should do something like `spring.cloud.azure.keyvault.mykeyvault` or `spring.cloud.azure.keyvaultmykeyvault`.